client/src/components/App.js

```diff
@@ -11,24 +11,43 @@ import ProfileInfoEdit from './profileInfoEdit/ProfileInfoEdit';
 
 import Register from './auth/Register';
 import Login from './auth/Login';
-import Header from './layout/Header'
+import Header from './layout/Header';
+import Sidebar from './layout/Sidebar';
+import { Container, Col, Row } from 'react-bootstrap';
+
+import './layout/Layout.css'
 
 import PublicationPage from './publications/PublicationPage';
+import { Fragment } from 'react';
 
 const App = () => {
+    const urls={
+        dashboard: "/dashboard",
+        profile: "/dashboard/profile"
+    }
```

Comment:
minor: this object doesn't seem to be adding much readability to the route paths. I'd say we can scrap it entirely or store each route path into this object as a constant.